tide/master/PixelStreamWindowManager.h

```diff
@@ -167,7 +167,7 @@ class PixelStreamWindowManager : public QObject
      *
      * @param uri the URI of the streamer
      */
-    void streamAdded(QString uri);
+    void externalStreamOpening(QString uri);
```

Comment:
The documentation should mention that at this point the stream window it hidden and the user is responsible to call showWindow() or handleStreamEnd().

Also, this class has unit tests and this new behaviour MUST be tested properly.